docs/ref/models/database-functions.txt

```diff
@@ -680,6 +680,565 @@ that deal with time-parts can be used with ``TimeField``::
     2014-06-16 00:00:00+10:00 2
     2016-01-01 04:00:00+11:00 1
 
+.. _math-functions:
+
+Math Functions
+==============
+
+.. versionadded:: 2.1
+
+We'll be using the following model in examples of math functions::
+ 
+    class Vector(models.Model):
+        x = models.FloatField()
+        y = models.FloatField()
+
+``Abs``
+-------
+
+.. class:: Abs(expression, **extra)
+ 
+Accepts a single numeric field or expression and returns the absolute value.
+ 
+Usage example::
+ 
+    >>> # Get the absolute value of the coordinates
+    >>> from django.db.models.functions import Abs
+    >>> Vector.objects.create(x=-0.5, y=1.1)
+    >>> vector = Vector.objects.annotate(x_abs=Abs('x'), y_abs=Abs('y')).get()
+    >>> print(vector.x_abs, vector.y_abs)
+    (0.5, 1.1)
+ 
+It can also be registered as a transform. For example::
+ 
+    >>> from django.db.models import FloatField
+    >>> from django.db.models.functions import Abs
+    >>> FloatField.register_lookup(Abs, 'abs')
```

Comment:
Similar to the tests, the second argument isn't needed, correct?

Reply:
Right. I have corrected all of the test cases.